src/app/child-dev-project/attendance/attendance-calendar/attendance-calendar.component.ts

```diff
@@ -143,6 +165,16 @@ export class AttendanceCalendarComponent implements OnChanges {
     await this.entityMapper.save(this.selectedEvent);
   }
 
+  newNote(): EventNote {
```

Comment:
To create a event for a activity the `createEventForActivity` function of the `AttendanceService` should be used.